demo/deploy-zookeeper.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
Running MySQL replication is a bit more complicated, and consensus doesn't happen automatically. The zookeeper helm chart configures ZK nodes to automatically find each other and perform leader election. This isn't meant to be a first-class demo feature yet; just a short-cut for folks who want to prove Statefulsets work

Reply:
It seems this would be more appropriate to be an e2e test, followed by a standalone stateful set demo at https://release-v1-1.docs.openservicemesh.io/docs/demos/

Reply:
E2E test [just landed](https://github.com/openservicemesh/osm/blob/168101f52c7f4f6df8b18cf07a9b798b9981b8f6/tests/e2e/e2e_statefulsets_test.go). Yeah, I definitely need to add a demo in there; should I add the demo before or after this PR?

Is there any harm in keeping this bash script in the repo though? It seems similar to the `deploy-vault.sh` script that's in the `main` branch: it doesn't get deployed in the regular demo, but it's there for convenience and displaying an OSM feature/integration.

Reply:
Yes having the script is fine. The demo can be added once the feature is ready.
My initial comment pertained to whether we could leverage the existing mysql stateful set. Alternatively, we could swap the mysql app with zookeeper in the default demo, if that's a better choice.

Reply:
If we really wanted to, we could invest the time to make MySQL truly HA for the current demo. I just didn't think that was the best move for right now since zookeeper is a simpler demo. Using Zookeeper instead of MySQL would be weird since Zookeeper isn't a DBMS, but I wouldn't be opposed to a pubsub demo with Kafka (which uses zookeeper) at a later date